modules/apps/object/object-admin-rest-impl/src/main/java/com/liferay/object/admin/rest/internal/resource/v1_0/ObjectDefinitionResourceImpl.java

```diff
@@ -267,6 +267,8 @@ private ObjectDefinition _toObjectDefinition(
 					}
 				).build();
 				active = objectDefinition.isActive();
+				categorization = objectDefinition.isCategorization();
```

Comment:
Since the story is not finished, I think that we should add these lines inside a condition+FF.
Inside this same method, there is an example of this with the property "storageType".
```
if (GetterUtil.getBoolean(PropsUtil.get("feature.flag.LPS-135430"))) {
	storageType = objectDefinition.getStorageType();
}
```
Do you agree?

Reply:
Yes, it makes sense